.teamcity/BuildParams.kt

```diff
@@ -4,7 +4,7 @@ object BuildParams {
   const val KOTLINX_COROUTINES_RELEASE_TAG = "1.9.0-RC"
   const val KOTLINX_SERIALIZATION_RELEASE_TAG = "v1.7.0-RC"
   const val KOTLINX_DATETIME_RELEASE_TAG = "v0.6.0"
-  const val KOTLINX_METADATA_JVM_RELEASE_TAG = "v0.9.0"
+  const val KOTLIN_RELEASE_TAG = "v2.0.0"
```

Comment:
Will it be updated automatically with every Kotlin release?

Reply:
We're working on it. I want to merge the changes to update the reference version to the latest one. The task [KTL-1399](https://youtrack.jetbrains.com/issue/KTL-1399/Align-kotlin-metadata-jvm-documentation-releases-with-Kotlin-releases) remains a work in progress.

Reply:
The idea is to fetch the latest release tag from Git, and use it for the reference.